public/index.html

```diff
@@ -14,6 +14,7 @@
     <!-- Default theme for CodeMirror: This will be changed later when we decide on a theme -->
     <link rel="stylesheet" href="bower_components/codemirror/theme/monokai.css" />
     <link href="css/style.css" rel="stylesheet"/>
+    <link type="text/css" rel="stylesheet" href="css/login.css">
```

Comment:
we should avoid doing separate stylesheets like this. We should try to make the styles more generic and put then inside style.css